osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -282,6 +290,13 @@ private void load(LocalisationManager localisation)
                     StatusPill.Hide();
             }
 
+            private static Drawable createStarRatingDisplay(BeatmapInfo beatmapInfo) => beatmapInfo.StarDifficulty > 0
+                ? new StarRatingDisplay(beatmapInfo)
+                {
+                    Margin = new MarginPadding { Bottom = 5 }
```

Comment:
Can we just use `Spacing` from `FillFlowContainer`?

`Spacing = new Vector2(0, 5),`

Reply:
Can't be used, because if the star rating is replaced with the `Empty`, the fill flow would add the spacing anyway.